crates/oxc_ast/src/ast_impl/js.rs

```diff
@@ -1303,7 +1303,7 @@ impl<'a> ClassElement<'a> {
         matches!(self, Self::StaticBlock(_))
     }
 
-    /// Returns `true` if this [`ClassElement`] is a static block or has a
+    /// Returns `true` if this [`ClassElement`] is a property and has a
     /// static modifier.
```

Comment:
Sorry to be a pain, but I don't think this is quite right either, because this suggests it has to be a property, not a method.

I suggest let's clear this up for once and for all!

```rs
/// Returns `true` if this [`ClassElement`] has a static modifier.
/// 
/// Note: Class static blocks do not have a "modifier", as there is no non-static equivalent.
/// Therefore, returns `false` for static blocks.
///
/// The following all return `true`:
/// ```ts
/// class {
///   static prop = 1;
///   static method() {}
///   static #private = 2;
///   static #privateMethod() {}
///   static accessor accessorProp = 3;
///   static accessor #privateAccessorProp = 4;
/// }
/// ```
```

Reply:
I'm too late. Merged while I was writing! Will make a follow-up PR.